src/game/default/g_types.h

```diff
@@ -482,8 +477,6 @@ typedef enum {
  * @brief Item types.
  */
 typedef enum {
-	ITEM_NONE,
-
```

Comment:
I ran into a little issue and realized that we were never even using the tag 0 for anything special, so it seemed wasteful to waste that index. While I was doing this to the tags, we also weren't using ITEM_NONE so I removed it as well - it would never be able to happen in-game so it seemed weird to have an enum for an impossibility.

If you think it's ambiguous I could change it, but when there's only one path possible (type > 0) then why have an enum entry for 0?